2023/06/15 15:59:13 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/06/15 17:08:47 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/06/15 17:45:48 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/06/25 23:29:57 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 14, in <module>
    from core.core import core
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\core.py", line 19, in <module>
    from core.scan import scan
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\scan.py", line 25, in <module>
    from payload.Exchange import Exchange
ModuleNotFoundError: No module named 'payload.Exchange'
)
2023/06/25 23:35:43 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/03 10:57:39 cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 14, in <module>
    from core.core import core
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\core.py", line 19, in <module>
    from core.scan import scan
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\scan.py", line 25, in <module>
    from payload.Exchange import Exchange
ModuleNotFoundError: No module named 'payload.Exchange'
)
2023/07/04 16:48:08 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/05 10:37:38 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/05 15:07:42 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/06 21:47:22 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/10 10:40:01 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 14, in <module>
    from core.core import core
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\core.py", line 19, in <module>
    from core.scan import scan
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\scan.py", line 25, in <module>
    from payload.Exchange import Exchange
ModuleNotFoundError: No module named 'payload.Exchange'
)
2023/07/10 10:41:13 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 14, in <module>
    from core.core import core
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\core.py", line 19, in <module>
    from core.scan import scan
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\scan.py", line 25, in <module>
    from payload.Exchange import Exchange
ModuleNotFoundError: No module named 'payload.Exchange'
)
2023/07/10 10:41:28 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/10 10:42:04 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 14, in <module>
    from core.core import core
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\core.py", line 19, in <module>
    from core.scan import scan
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\core\scan.py", line 25, in <module>
    from payload.Exchange import Exchange
ModuleNotFoundError: No module named 'payload.Exchange'
)
2023/07/10 14:13:33 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
2023/07/14 12:29:29 vulmapscan cmd.Run() failed with exit status 1
%!(EXTRA string=Traceback (most recent call last):
  File "vulmap.py", line 7, in <module>
    print(banner())  # 显示随机banner
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 187, in write_and_convert
    self.write_plain_text(text, cursor, start)
  File "D:\JDK\GDK\gopath\Aug\tools\vulmap\thirdparty\colorama\ansitowin32.py", line 195, in write_plain_text
    self.wrapped.write(text[start:end])
UnicodeEncodeError: 'gbk' codec can't encode character '\u2022' in position 194: illegal multibyte sequence
)
